Podcast/SeriesDetailViewController.swift

```diff
@@ -38,24 +38,26 @@ class SeriesDetailViewController: ViewController, SeriesDetailHeaderViewDelegate
         if let height = navigationController?.navigationBar.frame.maxY  {
             seriesHeaderViewY = height
         }
-        epsiodeTableView = UITableView(frame:  CGRect(x: 0, y: seriesHeaderViewY, width: view.frame.width, height: view.frame.height - seriesHeaderViewY))
-        epsiodeTableView.delegate = self
-        epsiodeTableView.dataSource = self
-        epsiodeTableView.tableHeaderView = seriesHeaderView
-        epsiodeTableView.showsVerticalScrollIndicator = false
-        epsiodeTableView.separatorStyle = .none
-        epsiodeTableView.addInfiniteScroll { (tableView) -> Void in
+        episodeTableView = UITableView(frame:  CGRect(x: 0, y: seriesHeaderViewY, width: view.frame.width, height: view.frame.height - seriesHeaderViewY))
```

Comment:
Can we add snap kit here?

Reply:
yes ofc but later, I am trying to limit the size of this PR